.circleci/config.yml

```diff
@@ -32,5 +32,5 @@ jobs:
           key: *cache_key
       - run: bazel run @nodejs//:yarn
       - run: bazel build ...
-      - run: bazel test ...
+      - run: bazel test ... --nolegacy_external_runfiles
```

Comment:
any comment about this flag, so that we know when is it safe to remove?

Reply:
Under current bazel, the external files are layed out in two places:
`runfiles_root/my_proj/external/npm/node_modules`
`runfiles_root/npm/node_modules`

They will eventually stop doing the first one. I added this flag to ensure that we are forwards compatible, opting into that breaking change. We could leave it here indefinitely.
Added a comment